### ISSUE TITLE
about preprocessed code

### ISSUE THREAD

**Issue by gilgamsh** [nachiket92/PGP#4]
when i want to run the third step
3. Run the following script to extract pre-processed data. This speeds up training significantly.
```shell
python preprocess.py -c configs/preprocess_nuscenes.yml -r path/to/nuScenes/root/directory -d path/to/directory/with/preprocessed/data
```
it turns out to be segmentation fault 
and i check that when  **from positional_encodings import PositionalEncodingPermute2D**
it come up with segmentation fault (core dumped) 

but when i import positional_encodings in the command line it turns out to be right.

**Comment by mengqingyu123** [comment]
I think this is probably caused by the mismatch between your tensorflow version and the cudatoolkit version. You can determine the matching relationship from https://www.tensorflow.org/install/source#common_installation_problems and reinstall.

**Comment by gilgamsh** [comment]
> I think this is probably caused by the mismatch between your tensorflow version and the cudatoolkit version. You can determine the matching relationship from https://www.tensorflow.org/install/source#common_installation_problems and reinstall.

Thank you very much. I switched the import sequence then get across it.

**Comment by weihuaLi1** [comment]
It can be solved by changing **"from positional_encodings import PositionalEncodingPermute2D"** to **"from positional_encodings.torch_encodings import PositionalEncodingPermute2D"**.

**Comment by amritpaul** [comment]
This worked for me-
**pip install --force-reinstall charset-normalizer==3.1.0**
